components/brave_wallet_ui/common/hooks/assets.ts

```diff
@@ -18,12 +18,24 @@ import usePricing from './pricing'
 import useBalance from './balance'
 import { useIsMounted } from './useIsMounted'
 import { useLib } from './useLib'
+import { httpifyIpfsUrl } from '../../utils/string-utils'
 
 const assetsLogo = (assets: BraveWallet.BlockchainToken[]) => {
-  return assets.map(token => ({
-    ...token,
-    logo: `chrome://erc-token-images/${token.logo}`
-  }) as BraveWallet.BlockchainToken)
+  return assets.map(token => {
+    let logo = token.logo
+    if (token.logo?.startsWith('ipfs://')) {
+      logo = httpifyIpfsUrl(token.logo)
```

Comment:
we would be loading HTTPS URLs directly here instead of via `chrome`. is that the expected behavior?

Reply:
i think if we are loading these logos in webui they should generally be passed through this sanitizer https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/ui/webui/sanitized_image_source.h?q=chrome:%2F%2Fimage&ss=chromium